content/en/docs/concepts/services-networking/service.md

```diff
@@ -14,44 +14,68 @@ weight: 10
 
 {{% capture overview %}}
 
-Kubernetes [`Pods`](/docs/concepts/workloads/pods/pod/) are mortal. They are born and when they die, they
-are not resurrected.  [`ReplicaSets`](/docs/concepts/workloads/controllers/replicaset/) in
-particular create and destroy `Pods` dynamically (e.g. when scaling out or in).  While each `Pod` gets its own IP address, even
-those IP addresses cannot be relied upon to be stable over time. This leads to
-a problem: if some set of `Pods` (let's call them backends) provides
-functionality to other `Pods` (let's call them frontends) inside the Kubernetes
-cluster, how do those frontends find out and keep track of which backends are
-in that set?
-
-Enter `Services`.
-
-A Kubernetes `Service` is an abstraction which defines a logical set of `Pods`
-and a policy by which to access them - sometimes called a micro-service.  The
-set of `Pods` targeted by a `Service` is (usually) determined by a [`Label
-Selector`](/docs/concepts/overview/working-with-objects/labels/#label-selectors) (see below for why you might want a
-`Service` without a selector).
-
-As an example, consider an image-processing backend which is running with 3
-replicas.  Those replicas are fungible - frontends do not care which backend
-they use.  While the actual `Pods` that compose the backend set may change, the
-frontend clients should not need to be aware of that or keep track of the list
-of backends themselves.  The `Service` abstraction enables this decoupling.
-
-For Kubernetes-native applications, Kubernetes offers a simple `Endpoints` API
-that is updated whenever the set of `Pods` in a `Service` changes.  For
-non-native applications, Kubernetes offers a virtual-IP-based bridge to Services
-which redirects to the backend `Pods`.
+{{< glossary_definition term_id="service" length="short" >}}
+
+No need to modify your application to use an unfamiliar service discovery mechanism.
+Kubernetes gives pods their own IP addresses and a single DNS name for a set of pods,
+and can load-balance across them.
 
 {{% /capture %}}
 
 {{% capture body %}}
 
+## Motivation
+
+Kubernetes {{< glossary_tooltip term_id="pod" text="Pods" >}}  are mortal.
+They are born and when they die, they are not resurrected.
+If you use a {{< glossary_tooltip term_id="deployment" >}} to run your app,
+it can create and destroy Pods dynamically (e.g. when scaling out or in).
+
+Each Pod gets its own IP address, however the set of Pods
+for a Deployment running in one moment in time could be different from
+the set of Pods running that application a moment later.
+
+This leads to a problem: if some set of Pods (call them “backends”) provides
+functionality to other Pods (call them “frontends”) inside your cluster,
+how do those frontends find out and keep track of which IP address to connect
+to, so that the frontend can use the backend part of the workload?
+
+Enter _Services_.
```

Comment:
i think we can take this one one

Reply:
?

Reply:
sorry, had a typo. Personally i don't think we need the text above

Reply:
@DanyC97 do you favour removing or rewording?

Reply:
@sftim i was in favour of removing it however i see folks are using it in blog posts so no longer a strong view on it ;)